src/Engine/Graphics/Direct3D/Render.cpp

```diff
@@ -1240,6 +1237,8 @@ void Render::SavePCXScreenshot() {
     SaveWinnersCertificate(file_name.c_str());
 }
 
+uint8_t *ReadScreenPixels() {}
```

Comment:
This looks out of place

Reply:
WDYM? I have to add dummies to DD renderer so it compile and we will be removing it eventually anyway...

Reply:
Ah OK, nevermind then